getting_started/dedicated_administrators.adoc

```diff
@@ -1,5 +1,15 @@
 [[getting-started-dedicated-administrators]]
 = Administering an {product-title} Cluster
+ifdef::openshift-dedicated[]
+:cluster-admin-name: Dedicated cluster administrator
+:cluster-admin-role: dedicated-cluster-admin
+:project-admin-role: dedicated-project-admin
+endif::[]
+ifdef::openshift-aro[]
+:cluster-admin-name: Customer cluster administrator
+:cluster-admin-role: customer-cluster-admin
+:project-admin-role: customer-project-admin
+endif::[]
```

Comment:
Not actually utilizing these attributes in this file yet because all the parts that would have been wrong for ARO have already been ifdef'd out, but adding these attributes here anyway in case for the future.